ui/app/components/secret-engine/configurable-secret-engine-details.hbs

```diff
@@ -0,0 +1,42 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{#if this.configError}}
+  {{! Surface API errors not associated with empty configuration details }}
+  <EmptyState data-test-config-error={{@model.id}} @title="Something went wrong" @message={{this.configError}} />
+{{else if this.configModel}}
+  {{#each this.configModel.attrs as |attr|}}
+    {{#if attr.options.sensitive}}
+      <InfoTableRow
+        alwaysRender={{(not (is-empty-value (get @model attr.name)))}}
+        @label={{or attr.options.label (to-label attr.name)}}
+        @value={{get this.configModel (or attr.options.fieldValue attr.name)}}
+      >
+        <MaskedInput @value={{get @model attr.name}} @name={{attr.name}} @displayOnly={{true}} @allowCopy={{true}} />
+      </InfoTableRow>
+    {{else}}
+      <InfoTableRow
+        @alwaysRender={{(not (is-empty-value (get @model attr.name)))}}
+        @label={{or attr.options.label (to-label attr.name)}}
+        @value={{get this.configModel (or attr.options.fieldValue attr.name)}}
+      />
+    {{/if}}
```

Comment:
I think you can wrap just the MaskedInput component in the `{{#if}}` block get the same results here which would remove the need for `{{else}}`
```hbs
<InfoTableRow
 alwaysRender={{(not (is-empty-value (get @model attr.name)))}}
 @label={{or attr.options.label (to-label attr.name)}}
 @value={{get this.configModel (or attr.options.fieldValue attr.name)}}
>
  {{#if attr.options.sensitive}}
    <MaskedInput @value={{get @model attr.name}} @name={{attr.name}} @displayOnly={{true}} @allowCopy={{true}} />
  {{/if}}
</InfoTableRow>
```

Reply:
I had made an inline comment about this but after several commits it was gone. Long story short, I can't. And I confirmed other instances of MaskedInput yielded inside of InfoTableRow. If I do, it does not render the non-masked Inputs. I've double checked this just to make sure I wasn't having an off day. Do a quick search for </InfoTableRow> and you'll see this pattern. I also thought of adding a conditional to InfoTableRow to handle Masked inputs, but defaulted to keeping scope focused on the files at hand.